adsingestp/parsers/datacite.py

```diff
@@ -96,15 +96,18 @@ def _parse_contrib(self, author=True):
                     or i.get("schemeURI", "") == "http://orcid.org"
                 ):
                     for ct in contrib_tmp:
-                        ct["orcid"] = i.get_text()
+                        try:
```

Comment:
Was this throwing an exception in testing? If not, reformulate as an `if` instead of try/except - something like this:
```
orcid = i.get_text().replace("https://orcid.org/","")
if orcid:
    ct["orcid"] = orcid
```

No need to do the else branch - any keys with a None value will get stripped out eventually anyway.

Reply:
I need to check this -- I think there was an exception thrown, but I'll verify.